Organisation.md

```diff
@@ -0,0 +1,46 @@
+# Organisationsform
+
+Organisationen, die autarke Teams etablieren wollen, haben diverse Topolgien zur Auswahl.
+
+Eine vielfältiger Überblick findet sich z.B. [hier](https://web.devopstopologies.com/).
+Die Orientierung gestaltet sich jedoch schwierig, der Austausch der Firmen im Roundtable hat ergeben, dass verschiedene dieser Topologien - mit unterschiedlichem Erfolg - erprobt wurden.
+
+## Idealbild
+
+Ein empfehlenswertes Idealbild könnte wie folgt aussehen:
+
+![DevOps Topology](devops-topo.svg)
+
+* **Dev** - Produkt-Team mit Fokus auf Umsetzung/Bereitstellung von Business-Funktionalität. Klassisch sind dies Software-Entwickler:innen ohne besonderen Betriebs- oder Infrastrukturhintergrund.
+* **DevOps** - Mitglieder des Produkt-Teams, die neben der Software-Entwicklung auch Infrastruktur-Tätigkeiten und Betrieb übernehmen. Idealerweise besteht das Produkt-Team ausschließlich aus diesem Profil, da dies jedoch utopisch ist, sollte eine ausreichende, mindestens jedoch redundaten Personenanzahl je Team vorhanden sein. Produkt-Teams dürfen und sollen zur Plattform beitragen.
+* **CCoE** (Cloud Center of Enablement) - Eine Gruppe von Cloud-Experten, die zwar dem Plattform-Team angehören, aber als Berater allen Teams mit Rat und Tat zur Seite stehen. Ihre primäre Aufgabe ist die Unterstützung der Produkt-Teams in der Umsetzung von Cloud-basierten Lösungen und des Aufbaus der DevOps-Rollen in jedem Team.
+* **Plattform** - Dediziertes Team, das den Produkt-Teams die sichere Verwendung der Cloud zugänglich macht. Dies erreichen sie u.a. dadurch, dass sie Werkzeuge für wiederkehrende Herausforderungen bereitstellen und allgemeine Qualitätsanforderungen in der Plattform verankern. Hierzu gehören typischerweise Aufgaben wie:
+  * Account-Verwaltung
+  * Governance ausüben
+  * Zentrale Kosten-Kontrolle
+  * Zentrale Dienste betrieben
+  * Allgemeingültiges Tooling bereitstellen
+Ein wichtiger Aspekt ist, dass die Plattform ein offenes, erweiterbares, kollaboratives System darstellt. Die Plattform ist die Basis, auf der die Produkt-Teams arbeiten, nicht die Grenze innerhalb der sie arbeiten.
+
+## Evolution
+
+Da die meisten Organisationen sich erst zum Idealbild hin entwickeln müssen, bedarf es Strategien, die diese Entwicklung unterstützen.
+Der Wechsel in eine DevOps-Organisationsform unter Verwendung von Cloud-Angeboten und -Technologie ist ein Prozess, der aktiv gefördert werden muss.
+
+Im Bestreben um autarke DevOps-Teams (also Teams, die ihr Produkt selbst designen, umsetzen, betreiben und pflegen) muss das Produkt-Team als Einheit, und nicht nur als Summe seiner Team-Mitglieder betrachtet werden.
```

Comment:
`DevOps-Teams` finde ich suboptimal als Bezeichnung. Wollen wir uns da an [Team Topologies](https://teamtopologies.com/key-concepts) anlehnen?

Vorschlag: `wertstromorientierte Produkt-Teams`

Reply:
find ich gut, übernehme ich gern wenn keine anderen Meinungen kommen.

Reply:
Wenn wir das anpassen, sollten wir aber den Betriebs-Aspekt in der Beschreibung ggf. noch etwas stärken. Der geht sonst schnell unter...